### PR TITLE
[Livechatv2] Add typing indicator api

### PR DESCRIPTION
**Pull request by aidhage**

### CODE REVIEW

File: src/Common/OmnichannelEndpoints.ts [hunk 1]
```diff
@@ -24,4 +24,5 @@ export default class OmnichannelEndpoints {
   public static readonly LiveChatGetReconnectableChatsPath = "livechatconnector/auth/reconnectablechats";
   public static readonly LiveChatReconnectAvailabilityPath = "livechatconnector/reconnect";
   public static readonly LiveChatValidateAuthChatMapRecordPath = "livechatconnector/auth/validateauthchatmaprecord";
+  public static readonly SendTypingIndicatorPath = "ccs-typingindicator/livechat/sendtypingindicator";
```
Comment (krishna0209) [inline src/Common/OmnichannelEndpoints.ts:27]: Why this pattern? It should just be the same as we have for incoming messages on LiveChatMessagingConnector

**Review by krishna0209 — APPROVED** [review]
Please ensure that the warning checks from GitHub actions are resolved prior to checking this in

File: src/Interfaces/ISDK.ts [hunk 2]
```diff
@@ -16,4 +16,5 @@ export default interface ISDK {
   emailTranscript(requestId: string, token: string, emailRequestBody: object): Promise<void>;
   fetchDataMaskingInfo(requestId: string): Promise<IDataMaskingInfo>;
   makeSecondaryChannelEventRequest(requestId: string, secondaryChannelEventRequestBody: object): Promise<void>;
+  sendTypingIndicator(requestid: string): Promise<void>;
```
Comment (xTEddie) [inline src/Interfaces/ISDK.ts:19]: nit: `requestId`  instead of `requestid`

File: src/SDK.ts [hunk 3]
```diff
@@ -955,4 +956,38 @@ export default class SDK implements ISDK {
       throw error;
     }
   }
+  
+  /** Send typing indicator
+   * @param requestId RequestId of the omnichannel session.
+   */
+  public async sendTypingIndicator(requestId:string): Promise<void> {
```
Comment (xTEddie) [inline src/SDK.ts:963]: I'd also add a guard to make sure this only works for v2

```
if (this.liveChatVersion !== LiveChatVersion.V2) { return new Error('Only supported on v2') }
```

File: src/Interfaces/ISDK.ts (hunk 2 above)
Comment (aidhage) [reply src/Interfaces/ISDK.ts]: updated

File: src/SDK.ts (hunk 3 above)
Comment (aidhage) [reply src/SDK.ts:963]: added this condition. THanks!

File: src/Common/OmnichannelEndpoints.ts (hunk 1 above)
Comment (aidhage) [reply src/Common/OmnichannelEndpoints.ts]: done. Update this api endpoint

File: src/SDK.ts [hunk 4]
```diff
@@ -955,4 +956,39 @@ export default class SDK implements ISDK {
...
```
Comment (xTEddie) [inline src/SDK.ts:963]: nit: Space after `:` 

`requestId: string`

Comment (aidhage) [reply src/SDK.ts]: dine

**Review by xTEddie — APPROVED** [review]

**Pull request merged**